go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/open-ibc/vib-core-smart-contracts/bindings
+module github.com/open-ibc/vib-core-smart-contracts/v2
```

Comment:
v2 might be confusing since that is a different number than our tagged versions?

Reply:
This has to be the major version only and you specify with a postfix of `"@v2.1.0"` or another full semantic version when doing go get/install/list etc the full version string [1]. The need to have a separate `/v2` directory is not needed as displayed by numerous packages and the go docs [2].

[1] https://go.dev/blog/v2-go-modules
[2] https://go.dev/wiki/Modules#releasing-modules-v2-or-higher

Reply:
what happens when we upgrade to major version v3? would we have to manually change this url then/remember to change it?

Reply:
Yeah the suffix would change to `/v3` CI would fail with a `v2` suffix and `v3.x.x` tag so will force us to remember when we do a tag release.